requirements.txt

```diff
@@ -40,6 +39,7 @@ mock==2.0.0
 ndg-httpsclient==0.4.2
 nose-pathmunge==0.1.2
 nose==1.3.3
+openpyxl==2.0.5
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `openpyxl` violates your company's policies.

Lifecycle recommends using version [2.1.3](https://pypi.org/project/openpyxl/2.1.3) instead as it does not violate any policies.